src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -37,6 +37,7 @@ namespace ngraph
             {
                 if (runtime::cpu::mkldnn_utils::use_mkldnn_kernel(node))
                 {
+                    auto qconv = static_cast<const ngraph::op::QuantizedConvolution*>(node);
```

Comment:
The compiler fails at this and below lines: https://aipg-rancher.intel.com/jenkins/algo/blue/organizations/jenkins/ngraph-ci-premerge/detail/PR-2351/1/pipeline/74#step-523-log-160